examples/gno.land/r/gh/gno.mod

```diff
@@ -0,0 +1 @@
+module gno.land/r/gh
```

Comment:
missing `require`?

```suggestion
module gno.land/r/gh

require "gno.land/p/demo/avl" v0.0.0-latest

```

Reply:
`gno lint` should complain?

anyways `gno mod tidy` implementation is complete. CI will start complaining about this soon.

Reply:
@harry-hov: We need a solution to eliminate those comments. The `gno mod` process should be fully automated, from creation to clarity in CI feedback. Please ensure it's streamlined so you don't have to review this aspect in future PRs. Otherwise, we might just bypass the need for gno.mod and switch to automatic detection.